src/commands/create.ts

```diff
@@ -1,66 +1,93 @@
 import { flags as oclifFlags } from '@oclif/command';
 import { isValidIdentifier } from '@superfaceai/ast';
-import { parseDocumentId } from '@superfaceai/parser';
+import {
+  DocumentVersion,
+  isValidDocumentIdentifier,
+  parseProfileId,
+} from '@superfaceai/parser';
 import { grey, yellow } from 'chalk';
 import inquirer from 'inquirer';
 
-import { Command } from '../common/command.abstract';
 import {
   composeUsecaseName,
+  DEFAULT_PROFILE_VERSION,
   DEFAULT_PROFILE_VERSION_STR,
-  inferCreateMode,
   SUPERFACE_DIR,
-} from '../common/document';
-import { CreateMode } from '../common/document.interfaces';
+} from '../common';
+import { Command } from '../common/command.abstract';
 import { developerError, userError } from '../common/error';
-import { mkdirQuiet } from '../common/io';
+import { exists, mkdirQuiet } from '../common/io';
 import { NORMALIZED_CWD_PATH } from '../common/path';
 import { create } from '../logic/create';
 import { initSuperface } from '../logic/init';
 import { detectSuperJson } from '../logic/install';
 
 export default class Create extends Command {
-  // hide the command from help
-  static hidden = true;
-
   static strict = false;
 
-  static description = 'Creates empty map and profile on a local filesystem.';
-
-  static args = [
-    {
-      name: 'documentInfo',
-      required: true,
-      description:
-        'Two arguments containing informations about the document.\n1. Document Type (optional) - type of document that will be created (profile or map), if not specified, utility will create both\n2. Document Name - name of a file that will be created',
-    },
-  ];
+  static description =
+    'Creates empty map, profile or/and provider on a local filesystem.';
 
   static flags = {
     ...Command.flags,
+    //Inputs
+    profileId: oclifFlags.string({
+      description: 'Profile Id in format [scope](optional)/[name]',
+    }),
+    providerName: oclifFlags.string({
```

Comment:
I would add more dependencies between flags, e.g. `--providerName` depends on `--map` or `--provider` or that `--variant` depends on `--map` and so on.

Reply:
It won't work because of interactive mode. :/